cldf/cldf-metadata.json

```diff
@@ -23,15 +23,15 @@
         {
             "dc:title": "environment",
             "properties": {
-                "glottolog_version": "v4.0-12-ge90a2e37f4",
-                "concepticon_version": "v2.0-30-gcea4c63"
+                "glottolog_version": "v4.0-12-ge90a2e3",
```

Comment:
I guess you run this against `HEAD`, not an explicitly checked out version, right? I think it would be better to do
```
git checkout v4.0
```
explicitly.